### PR TITLE
Merge webp and webp_replace img_optm settings

### PR DESCRIPTION
**Pull request by tynanbe**

### CODE REVIEW

File: tpl/img_optm/settings.tpl.php [hunk 1]
```diff
@@ -107,17 +108,13 @@
...
+				<br /><?php Doc::notice_htaccess(); ?>
+				<br /><?php Doc::crawler_affected(); ?>
 			</div>
 		</td>
 	</tr>
-
-	<?php
-		if ( ! is_multisite() ) :
-			// webp
-			require LSCWP_DIR . 'tpl/img_optm/settings.media_webp.tpl.php';
```
Comment (hi-hai) [inline tpl/img_optm/settings.tpl.php:117]: The reason we use separate file is for network settings as well. If you remove this file, in `img_optm/network_settings.tpl.php`, there will be no this option

Comment (tynanbe) [reply tpl/img_optm/settings.tpl.php]: Thanks, missed that. Should be fixed now.

File: tpl/general/settings.tpl.php [hunk 2]
```diff
@@ -195,12 +195,6 @@
 				⚠️ <?php echo __( 'Notice', 'litespeed-cache' ); ?>: <?php echo sprintf( __( 'You need to turn %s on and finish all WebP generation to get maximum result.', 'litespeed-cache' ),  '<code>' . Lang::title( Base::O_IMG_OPTM_WEBP ) . '</code>' ); ?>
 				</font>
 				<?php endif; ?>
-
-				<?php if ( ! $this->conf( Base::O_IMG_OPTM_WEBP_REPLACE ) ) : ?>
-				<br /><font class="litespeed-danger litespeed-left10">
-				⚠️ <?php echo __( 'Notice', 'litespeed-cache' ); ?>: <?php echo sprintf( __( 'You need to turn %s on to get maximum result.', 'litespeed-cache' ),  '<code>' . Lang::title( Base::O_IMG_OPTM_WEBP_REPLACE ) . '</code>' ); ?>
-				</font>
-				<?php endif; ?>
```
Comment (hi-hai) [inline tpl/general/settings.tpl.php:203]: I think we still need to give this notice

Comment (tynanbe) [reply tpl/general/settings.tpl.php:203]: The notice is still given immediately above.

**Pull request merged**